### ISSUE TITLE
[BUG] "tiddlywiki +extraPlugins invalid-path --listen" causes runtime error. It should return an error message

### ISSUE THREAD

**Issue by pmario** [TiddlyWiki/TiddlyWiki5#8543]
To reproduce

The following commands work:

```
tiddlywiki test --init empty
tiddlywiki +plugins/tiddlywiki/filesystem +plugins/tiddlywiki/tiddlyweb empty --listen
```

The following command **throws an error** -- It should create an Error message

```
tiddlywiki +plugins/tiddlywiki/filesystem +plugins/tiddlywiki/tiddlyweb emptyAAAA --listen
```

creates (output shortened for better readability)

```
> tiddlywiki +plugins/tiddlywiki/filesystem +plugins/tiddlywiki/tiddlyweb emptyXX --listen
$:/core/modules/utils/filesystem.js:114
        if(dirPath.substr(dirPath.length-1,1) !== path.sep) {
                   ^

TypeError: Cannot read properties of undefined (reading 'substr')
    at exports.createDirectory ($:/core/modules/utils/filesystem.js:114:13)
    at new FileSystemAdaptor ($:/plugins/tiddlywiki/filesystem/filesystemadaptor.js:25:12)
    at $:/core/modules/startup.js:124:22
...
Node.js v20.14.0
```

**expected**

```
> tiddlywiki +plugins/tiddlywiki/filesystem +plugins/tiddlywiki/tiddlyweb emptyXX --listen

Error: "emptyXX" does not exist 
```

or something similar. 

-------

Remark: I know what causes the problem. This issue is just a reminder.

**Comment by Jermolene** [comment]
Thanks @pmario fixed in 38081b86c97a795420515156fcd52177574be516

**Comment by pmario** [comment]
Thx for the fast fix.
